### ISSUE TITLE
AVT 1 - Carbon X Overflow Menu has DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1774]
macOS Mojave version 10.14.2, Chrome version 71.0.3578.98 (Official Build) (64-bit), DAP w/ latest rule set Aug. 2018 

Run DAP on Overflow Menu with the menu button pressed so that the contents of the menu displays. (See screenshot for example and errors).
<img width="1092" alt="screen shot 2019-02-11 at 3 10 06 pm" src="https://user-images.githubusercontent.com/21676914/52595889-983f0000-2e14-11e9-90df-1809fd46948d.png">

Note: By adding aria-disabled="true" to the disabled menu item will the contrast violation.
